admin-dev/themes/new-theme/js/pages/product/product-map.js

```diff
@@ -33,6 +33,7 @@ export default {
     productSuppliersCollection: `${productSuppliersId}`,
     supplierIdsInput: '#product_suppliers_supplier_ids',
     defaultSupplierInput: '#product_suppliers_default_supplier_id',
+    // @todo: why its called productsTable and not productSuppliers|suppliers table? is it a mistake?
```

Comment:
Because it's already in a "sub namespace" `suppliers`, also it was to avoid having selector names too long (because of ESlint columns restriction)

But with the latest changes maybe the name are not correct any more and should be improved Besides the ESlint limitation could be tricked by using only the sub namespace

```
import ProductMap from `ProductMap`;
const SuppliersMap = Product.suppliers;
```

I would have preferred to use object destructuring but it doesn't seem to be that easy with import.

Reply:
I find productsTable misleading, like if suppliers would contain a table of its products or so.. Im ok with just a "table", as you said its under "suppliers" already so its clear.